### PR TITLE
WRR-10798: Fixed redundant variables, missing await, redundant await, deprecated functions

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed redundant variables, missing await, redundant await, deprecated functions.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-10798

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]

File: Dropdown/DropdownList.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ const DropdownListBase = kind({
 	},
 
 	computed: {
-		className: ({direction, skin, width, styler}) => styler.append(direction.substr(0, direction.indexOf(' ')), width, {dropdownListWithCustomizedScroller: skin === 'silicon'}),
+		className: ({direction, skin, width, styler}) => styler.append(direction.substring(0, direction.indexOf(' ')), width, {dropdownListWithCustomizedScroller: skin === 'silicon'}),
```
Comment (seunghoh) [inline Dropdown/DropdownList.js:144]: OK

File: Item/Item.module.less [hunk 2]
```diff
@@ -231,7 +231,7 @@
 				min-width: @agate-item-indicator-height; // `min-width` is used to ensure that the indicator has a correct width with default item
 				background-color: @agate-item-indicator-bg-color;
 				border-radius: @agate-item-indicator-border-radius;
-				align-self: '@agate-item-indicator-align-self';
+				align-self: @agate-item-indicator-align-self;
```
Comment (seunghoh) [inline Item/Item.module.less:234]: OK

File: ThemeDecorator/ThemeDecorator.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
 import classnames from 'classnames';
 import convert from 'color-convert';
 import PropTypes from 'prop-types';
-import {createContext, Component} from 'react';
+import {Component, createContext} from 'react';
```
Comment (seunghoh) [inline ThemeDecorator/ThemeDecorator.js:20]: OK

File: tests/ui/specs/MediaPlayer/MediaPlayer-specs.js [hunk 4]
```diff
@@ -111,7 +111,7 @@ describe('MediaPlayer', function () {
 			expect((await mediaPlayerDisabled.knob.getCSSProperty('left')).value).toBe('0px');
 
 			await mediaPlayerDisabled.playButton.click();
-			Page.delay(1000);
+			await Page.delay(1000);
```
Comment (seunghoh) [inline tests/ui/specs/MediaPlayer/MediaPlayer-specs.js:114]: OK

File: tests/ui/specs/MediaPlayer/MediaPlayer-specs.js [hunk 5]
```diff
@@ -120,7 +120,7 @@ describe('MediaPlayer', function () {
 			expect(await mediaPlayerDisabled.source()).toBe('https://sampleswap.org/mp3/artist/254731/BossPlayer_Your-Right-Here-160.mp3');
 
 			mediaPlayerDisabled.nextButton.click();
-			Page.delay(1000);
+			await Page.delay(1000);
```
Comment (seunghoh) [inline tests/ui/specs/MediaPlayer/MediaPlayer-specs.js:123]: OK

File: tests/ui/specs/ToggleButton/ToggleButton-Page.js [hunk 6]
```diff
@@ -8,7 +8,7 @@ class AgateToggleButtonInterface {
 	}
 
 	async focus () {
-		return browser.execute((el) => el.focus(), await `#${this.id}`);
+		return browser.execute((el) => el.focus(), `#${this.id}`);
```
Comment (seunghoh) [inline tests/ui/specs/ToggleButton/ToggleButton-Page.js:11]: OK

File: tests/ui/specs/RangePicker/RangePicker-specs.js [hunk 7]
```diff
@@ -116,7 +116,7 @@ describe('RangePicker', function () {
 				it('should increase to min value', async function () {
 					for (let i = 0; i < 10; i++) {
 						await rangePicker.incrementer(rangePicker.self).click();
-						await browser.pause(100);
+						await browser.pause(500);
```
Comment (seunghoh) [inline tests/ui/specs/RangePicker/RangePicker-specs.js:119]: Would you explain why we should change 100 -> 500?

File: tests/ui/specs/VirtualList/VirtualList-utils.js [hunk 8]
```diff
@@ -11,7 +11,7 @@ async function hitTest (_selector) {
 			targetRect = await target.getBoundingClientRect(),
 			targetDown = [targetRect.x + (targetRect.width / 2), targetRect.y + targetRect.height - 1],
 			targetTop = [targetRect.x + (targetRect.width / 2), targetRect.y + 1];
-		return await target.contains(document.elementFromPoint(...targetDown)) || target.contains(document.elementFromPoint(...targetTop));
+		return target.contains(document.elementFromPoint(...targetDown)) || target.contains(document.elementFromPoint(...targetTop));
```
Comment (seunghoh) [inline tests/ui/specs/VirtualList/VirtualList-utils.js:14]: OK

**Review by seunghoh — CHANGES_REQUESTED** [review]
Thanks for the PR, I understand your changes but I'd prefer the original code from a readability perspective. I had a similar discussion with  @daniel-stoian-lgp before. Let's fix only the obvious fault codes. I commented `OK` inline.

**Review by seunghoh — APPROVED** [review]
LGTM

**Pull request merged**